dataactvalidator/scripts/load_cfda.py

```diff
@@ -0,0 +1,16 @@
+from ftplib import FTP
+
+ftp = FTP('ftp.cfda.gov')      # connect to host, default port
+ftp.login()                    # user anonymous, passwd anonymous@
+ftp.cwd('usaspending')
+
+data = []
+ftp.dir('-t', data.append)
+file_listing = data[0]
+file_parts = file_listing.split(' ')
+file_name = file_parts[-1]
+print('Loading ' + file_name)
+
+ftp.retrbinary('RETR ' + file_name, open('dataactvalidator/config/cfda_program.csv', 'wb').write)
```

Comment:
Here, we may want to consider streaming directly to the database versus downloading the data, writing to the file, and then processing that new file. 

This could potentially take up a lot of storage as well so if we decide to stay with this current process, the file should be deleted once it's been loaded into the database.